components/prism-php.js

```diff
@@ -149,64 +172,112 @@
 		inside: Prism.languages.php
 	};
 
-	Prism.languages.insertBefore('php', 'variable', {
-		'string': [
-			{
-				pattern: /<<<'([^']+)'[\r\n](?:.*[\r\n])*?\1;/,
-				alias: 'nowdoc-string',
-				greedy: true,
-				inside: {
-					'delimiter': {
-						pattern: /^<<<'[^']+'|[a-z_]\w*;$/i,
-						alias: 'symbol',
-						inside: {
-							'punctuation': /^<<<'?|[';]$/
-						}
+	var string = [
+		{
+			pattern: /<<<'([^']+)'[\r\n](?:.*[\r\n])*?\1;/,
+			alias: 'nowdoc-string',
+			greedy: true,
+			inside: {
+				'delimiter': {
+					pattern: /^<<<'[^']+'|[a-z_]\w*;$/i,
+					alias: 'symbol',
+					inside: {
+						'punctuation': /^<<<'?|[';]$/
 					}
 				}
-			},
-			{
-				pattern: /<<<(?:"([^"]+)"[\r\n](?:.*[\r\n])*?\1;|([a-z_]\w*)[\r\n](?:.*[\r\n])*?\2;)/i,
-				alias: 'heredoc-string',
-				greedy: true,
-				inside: {
-					'delimiter': {
-						pattern: /^<<<(?:"[^"]+"|[a-z_]\w*)|[a-z_]\w*;$/i,
-						alias: 'symbol',
-						inside: {
-							'punctuation': /^<<<"?|[";]$/
-						}
-					},
-					'interpolation': string_interpolation // See below
-				}
-			},
-			{
-				pattern: /`(?:\\[\s\S]|[^\\`])*`/,
-				alias: 'backtick-quoted-string',
-				greedy: true
-			},
-			{
-				pattern: /'(?:\\[\s\S]|[^\\'])*'/,
-				alias: 'single-quoted-string',
-				greedy: true
-			},
-			{
-				pattern: /"(?:\\[\s\S]|[^\\"])*"/,
-				alias: 'double-quoted-string',
-				greedy: true,
-				inside: {
-					'interpolation': string_interpolation // See below
+			}
+		},
+		{
+			pattern: /<<<(?:"([^"]+)"[\r\n](?:.*[\r\n])*?\1;|([a-z_]\w*)[\r\n](?:.*[\r\n])*?\2;)/i,
+			alias: 'heredoc-string',
+			greedy: true,
+			inside: {
+				'delimiter': {
+					pattern: /^<<<(?:"[^"]+"|[a-z_]\w*)|[a-z_]\w*;$/i,
+					alias: 'symbol',
+					inside: {
+						'punctuation': /^<<<"?|[";]$/
+					}
+				},
+				'interpolation': string_interpolation // See below
+			}
+		},
+		{
+			pattern: /`(?:\\[\s\S]|[^\\`])*`/,
+			alias: 'backtick-quoted-string',
+			greedy: true
+		},
+		{
+			pattern: /'(?:\\[\s\S]|[^\\'])*'/,
+			alias: 'single-quoted-string',
+			greedy: true
+		},
+		{
+			pattern: /"(?:\\[\s\S]|[^\\"])*"/,
+			alias: 'double-quoted-string',
+			greedy: true,
+			inside: {
+				'interpolation': string_interpolation // See below
+			}
+		}
+	];
+
+	Prism.languages.insertBefore('php', 'variable', {
+		'string': string,
+	});
+
+	Prism.languages.insertBefore('php', 'variable', {
+		'attribute': {
+			pattern: /#\[(?:(?!#\[)(?:[^"'\/#]|\/\/.*$|#.*$|\/\*(?:[^*]|\*(?!\/))*\*\/|"(?:\\[\s\S]|[^\\"])*"|'(?:\\[\s\S]|[^\\'])*'))+](?=\s*[a-z$#])/mi,
```

Comment:
1. The alternative for division `\/(?![*/])` was missing.
1. `(?!#\[)` can be moved inside the alternation. It only affects the `#.*$` alternative, so that's the only one that has to be changed.

Also, please always escape `]` characters.

```suggestion
			pattern: /#\[(?:[^"'\/#]|\/(?![*/])|\/\/.*$|#(?!\[).*$|\/\*(?:[^*]|\*(?!\/))*\*\/|"(?:\\[\s\S]|[^\\"])*"|'(?:\\[\s\S]|[^\\'])*')+\](?=\s*[a-z$#])/mi,
```